### ISSUE TITLE
Problems with pipeline.fit in DeepLearning_ImageClassification_Training

### ISSUE THREAD

**Issue by SidV256** [dotnet/machinelearning-samples#663]
Initially I was able to train and run your default dataset of flowers with no error and was able to train my own data as well but while building model I am facing a variety of errors including:
_System.FormatException: 'Tensorflow exception triggered while loading model.'
InvalidProtocolBufferException: While parsing a protocol message, the input ended unexpectedly in the middle of a field.  This could mean either that the input has been truncated or that an embedded message misreported its own length._

OR

_failed to create a newwriteablefile: : the system cannot find the path specified._

Please help me I am using ResnetV2101 as my architecture because InceptionV3 throws errors even before this line.

**Comment by raskolnikoov** [comment]
@SidV256 I have also run into this issue. Can't seem to get it to work. :/

**Comment by luisquintanilla** [comment]
@SidV256 @raskolnikoov Sorry to hear you're having trouble. Not sure if this might help, but this issue has been logged in the ML.NET repo dotnet/machinelearning#4234 . 

I'm not sure if it was solved but I was able to train and the error did not appear again given the pipeline I provided. I would also suggest cleaning the solution (deleting the /bin /obj directories in the project) and running again.
